HSLdevcom/jore-map-ui
# 457 edit network clicking network node

Closes #457 

* Network's layers are now permanently set as interactive (because you can't change that value once it is declared). Mentioned about it here: #447
   * Also layers need to be redrawn to be hidden (didn't end up with a better solution)
* Added TransitType to INode because I thought that I would have to use it here -> was wrong. However, we need this info if we implement this task: #471
* Also finally found a solution to the double click event bug. Fixed it in this pr.

How to test:

1. At network mode, activate "edit network node" tool
1. Select a node from map
1. Network nodes & links should be highlighted belonging to selected node. Network's links & nodes should be hidden
1. Try to refresh page, the same selection should be highlighted

-> focusing map bounds to the highlighted links: new card: #470
